content/en/docs/security/otel-collector-configuration-best-practices.md

```diff
@@ -0,0 +1,147 @@
+---
+title: OpenTelemetry Collector Configuration Best Practices
+description:
+  Follow best practices to securely configure OpenTelemetry Collector.
+weight: 20
+---
+
+When setting up configuration for OpenTelemetry (OTel) Collector, consider the
+following practices to better secure your OTel Collector instance.
+
+## Configuration
+
+<!--
+TODO: SHOULD only enable the minimum required components. What are those?
+--->
+
+## Receivers and Exporters
+
+- Receivers and Exporters can be either push or pull-based. In either case, the
+  connection established SHOULD be over a secure and authenticated channel.
+- Unused receivers and exporters SHOULD be disabled to minimize the attack
+  vector of the Collector.
+- Receivers and Exporters MAY expose buffer, queue, payload, and/or worker
```

Comment:
Thank you, @jpkrohling! For values that are too wide, rather than:
> remember that values that are too wide (such as a large queue number) might mean that the Collector will attempt to use more memory than a setting with a smaller value.

Would it be more accurate to describe as: 
> remember that if the values are too wide, such as a large queue number, then the OTel Collector may use more memory than necessary, compared to using a setting with a smaller value.

Reply:
@jpkrohling Circling back to ask your feedback on the above ^